.kitchen.yml

```diff
@@ -0,0 +1,33 @@
+---
+driver:
+  name: vagrant
+
+provisioner:
+  name: chef_zero
+  require_chef_omnibus: 12.9
+
+platforms:
+  - name: centos-7.2
+  - name: debian-8.4
+  - name: windows
+    driver:
+      box: win2012r2-standard-nocm-1.1.0
+      box_url: http://repo.release.cerner.corp/nexus/content/repositories/vagrant/com/cerner/vagrant/win2012r2-standard-nocm/1.1.0/win2012r2-standard-nocm-1.1.0.box
```

Comment:
This isn't public, so we can't use it. instead what about using a box of opentable/win-2012r2-standard-amd64-nocm 

https://atlas.hashicorp.com/opentable/boxes/win-2012r2-standard-amd64-nocm/